wemake_python_styleguide/logic/tree/classes.py

```diff
@@ -34,3 +38,76 @@ def is_forbidden_super_class(class_name: Optional[str]) -> bool:
     if class_name in ALLOWED_BUILTIN_CLASSES:
         return False
     return is_builtin_name(class_name)
+
+
+def get_attributes(
+    node: ast.ClassDef,
+    include_annotations: bool,
+) -> Tuple[List[types.AnyAssign], List[ast.Attribute]]:
+    """Returns all non annotated class and instance attributes of a class."""
+    instance_attributes = get_instance_attributes(node)
+    class_attributes = []
+    if include_annotations:
+        class_attributes = get_annotated_class_attributes(node)
+    else:
+        class_attributes = get_class_attributes(node)
+    return class_attributes, instance_attributes
+
+
+def get_class_attributes(
+    node: ast.ClassDef,
+) -> List[Union[ast.Assign, ast.AnnAssign]]:
+    """Returns all non annotated class attributes of a class."""
+    class_attributes = []
+    for sub in ast.walk(node):
+        correct_context = nodes.get_context(sub) == node
```

Comment:
what do you think about renaming `correct_context` to `is_context_correct`?
It sounds more proper for boolean and then next `if` statement can be read like "plain english" sentence

Reply:
Sure!